test/server.js

```diff
@@ -0,0 +1,154 @@
+'use strict'
```

Comment:
We don't use this server anymore. Take the new one from appc.mysql or appc.mssql projects